Fixed Serbian translation

ljubar opened the pull request:
- fixed expression for different values of %count
- removed old file with mis-spelled nname

weaverryan commented:
Hey @ljubar!

Thanks for this! However, I need some help figuring out the removed translation situation. Basically:

A) `sr_Latn.xliff` was renamed to `sr_Latin.xliff` 3 years ago sha: a8f1c3bf95d4b6ed02e69073542e9f27ef33b2ba

B) A new `sr_Latn.xliff` as created about 2 years ago #76 

Also, FOSUserBundle only has the `Latn` versions https://github.com/FriendsOfSymfony/FOSUserBundle/tree/master/Resources/translations

So, do we simply have a translation file duplicated? It appears to me that there are various differences. Which is better? My Serbian isn't very good ;) - so I need some help determining what best to do.

Thanks!

ljubar commented:
Hey @weaverryan -
I thought 'Latn' was misspelled 'Latin' :)
Yes there were two files: sr_latn.xliff and sr_latin.xliff - I deleted the former and corrected the latter as it was more advanced (covering more precisely the cases for different numbers). That one had issue with angled brackets.
I wasn't testing with locale active, I simply overrode English version with my new file for development purposes of the project I work on. Now that corrected version works fine.
So should I create new PR, or you can cherry-pick somehow?
Best, Ljuba

weaverryan commented:
Sorry for dropping this. As far as I can tell, either latn or latin should be valid - it kind of seems like it was maybe latn originally... and maybe some people have changed to latin.

Either way, we only need one. I'm going to merge your changes.

Thanks!
